Documentation/ColumnsConfig/Type/Group/_Properties/_FieldControl.rst.txt

```diff
@@ -88,13 +118,34 @@
         button to open an element browser. It is enabled by default if rendering a
         group element.
 
+        ..  confval:: renderType
+            :name: group-fieldControl-elementBrowser-renderType
```

Comment:
does this need to be all lowercase or not?

Reply:
name needs to be lowercase

Reply:
is that really true? I have found others to be in the same format.